Sources/Subscription/Flows/AppStore/AppStorePurchaseFlow.swift

```diff
@@ -108,14 +110,41 @@ public final class AppStorePurchaseFlow {
             }
         }
     }
-    // swiftlint:enable cyclomatic_complexity
 
+    // swiftlint:enable cyclomatic_complexity
     @discardableResult
-    public static func completeSubscriptionPurchase() async -> Result<PurchaseUpdate, AppStorePurchaseFlow.Error> {
+    public static func completeSubscriptionPurchase(with transactionJWS: TransactionJWS) async -> Result<PurchaseUpdate, AppStorePurchaseFlow.Error> {
         os_log(.info, log: .subscription, "[AppStorePurchaseFlow] completeSubscriptionPurchase")
 
-        let result = await AccountManager.checkForEntitlements(wait: 2.0, retry: 20)
+        guard let accessToken = AccountManager().accessToken else { return .failure(.missingEntitlements) }
+
+        let result = await callWithRetries(wait: 0.5, retry: 3) {
```

Comment:
Isn't this wait too short?  Why only 3 retries?

Reply:
What are your suggestions?